### PR TITLE
Fixed location radio button position

### PR DESCRIPTION
**Pull request by donaldkibet**
Screenshot from 2020-02-03 22-17-17

### CODE REVIEW

File: src/choose-location/choose-location.component.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function ChooseLocation(props: ChooseLocationProps) {
         .catch(createErrorHandler());
     }
     return () => abortController.abort();
-  }, [isSubmitting]);
+  }, [config.links.loginSuccess, isSubmitting, location, props]);
```
Comment (brandones) [inline src/choose-location/choose-location.component.tsx]: I'm asking about this in https://twitter.com/brandard_err/status/1223295010122141696 ... I'm not sure we should change these? This calls the handler every time any one of these four things changes, right?

Comment (brandones) [reply src/choose-location/choose-location.component.tsx]: Can we make this PR just about fixing the placement, and split the useEffect dependency stuff into another PR? I really wanna approve the placement stuff quick.

**Comment by brandones** [comment]
@donaldkibet Thanks for doing this! Left some feedback.

attn @mseaton

**Comment by mseaton** [comment]
@donaldkibet this seems like it might not be up-to-date with the latest master of openmrs-esm-login.  If it were, I'd expect that if only one location were present that it would be skipped.  Also, from what I recall of openmrs-spa, there are multiple locations - in your screenshot above there is only one.  So I'm guessing there are still issues to resolve to get the rest to appear.  Also, this screenshot doesn't reflect the new radio button style that was committed.

For me the biggest reason why this is broken is because it was developed off of the latest release of openmrs-esm-styleguide.  It would be great if we could release more often so that we can develop against the latest versions.

File: src/choose-location/choose-location.component.tsx (hunk 1 above)
Comment (donaldkibet) [reply src/choose-location/choose-location.component.tsx]: > Can we make this PR just about fixing the placement, and split the useEffect dependency stuff into another PR? I really wanna approve the placement stuff quick.

Agree will fix the useEffect dependency with another PR.

**Comment by donaldkibet** [comment]
> @donaldkibet this seems like it might not be up-to-date with the latest master of openmrs-esm-login. If it were, I'd expect that if only one location were present that it would be skipped. Also, from what I recall of openmrs-spa, there are multiple locations - in your screenshot above there is only one. So I'm guessing there are still issues to resolve to get the rest to appear. Also, this screenshot doesn't reflect the new radio button style that was committed.
> 
> For me the biggest reason why this is broken is because it was developed off of the latest release of openmrs-esm-styleguide. It would be great if we could release more often so that we can develop against the latest versions.

The reason why only one location was being displayed was with the CSS. `display: flex` property in `.location-radio-group` class. I have it fixed and is working as expected.

**Comment by mseaton** [comment]
@donaldkibet - sorry, I'm not sure if you saw my comment above from yesterday?  This commit basically reverts the work I did in an earlier commit.  What I think we need to do is fix things so that my previous work is compatible with the latest styleguide, as it was developed against the latest _release_ of the styleguide (1.2), not master.  

Can we please revert this until we resolve the issue?

**Pull request merged**